packages/v4/src/content/design-guidelines/components/page/page.md

```diff
@@ -2,4 +2,58 @@
+
+### Page with horizontal navigation
+
+<img src="./img/page-horizontal-nav.png" alt="page with horizontal navigation" width="832"/>
+
+When using a horizontal primary navigation, the left-hand sidebar is removed and the navigation is placed inline with other masthead elements. All other page elements remain the same.
+
+## Masthead
+The masthead contains global properties such as logotype, navigation and settings in an organized fashion and it is accessible from all parts of the application. All pages should share a common masthead.
+
+<img src="./img/masthead.png" alt="masthead" width="832"/>
+
+**1. Show/hide menu:**  provides a way for users to toggle vertical navigation. We recommend always having the menu icon visible when using vertical navigation. Show/hide menu is also used when application is being displayed in smaller viewport sizes, and it should stay visible in smaller viewport sizes.
+
+**2. Logotype:** The logotype area should clearly display the application name and the application logo if applicable. It is common to link the logotype to the application’s homepage. It is recommended to have the application name in .svg format for better performance.
+
+**3. Utility items:** Place any items in the utility area that you want to have accessible from all pages in the application. Utility items are right aligned after the user menu. The following are common items ordered from left to right. All items are optional depending on the needs of your application.
+
+* **Notification badge:** Display a [notification badge](/components/notification-badge) to alert the user to incoming notifications. Clicking on the badge should open a [notification drawer](/components/notification-drawer), if used, or take the user to another view where they can review recent notifications. If used, the notification badge should always be the leftmost item in the utility area.
+
+* **Application specific item(s):** Applications may include additional utility items in the masthead. These items should expose tasks or information that are meaningful to users at a global level.  
+
+* **Application launcher:** When switching between applications is supported, an [Application launcher](/components/application-launcher) component can be inserted in the masthead to provide that capability.
+
+* **Settings:**  The settings menu contains ways to configure and customize an application. Interacting with settings either takes users to a settings area or reveals a dropdown menu with more options. The settings menu is represented by the `fa-cog` icon.
+
+* **Help:** The help item reveals a dropdown menu. The menu items may include documentation links available to users, links to customer support resources, and/or instructions on how to get support. If the application has an [About modal](/components/about-modal), it should also be accessed here . The help menu is represented by the `pficon-pf-help` icon.
```

Comment:
Extra space between "it should also be accessed here" and the period after it!

Reply:
I also think that "application launcher" and "about modal" should be sentence case when referred to in the sentences above

Reply:
I don't see this. Aren't they already sentence case?

Reply:
No, the first word is capitalized when it shouldn't be. So I mean change "About modal" to "about modal"

Reply:
Ah, OK. I misunderstood.